minlzlib/CMakeLists.txt

```diff
@@ -18,6 +18,6 @@ if(MSVC)
     set(CMAKE_C_FLAGS_RELWITHDEBINFO "/Ox /Ob2 /Oi /Ot /Oy /GF /Gy /MT /Zi /wd4214 /permissive-")
     set(CMAKE_SHARED_LINKER_FLAGS_RELWITHDEBINFO "/OPT:ICF /OPT:REF /NODEFAULTLIB /NOENTRY /DEBUG /EXPORT:XzDecode /EMITPOGOPHASEINFO /NOVCFEATURE /NOCOFFGRPINFO /FILEALIGN:512 /DRIVER /MANIFEST:NO /PDBALTPATH:minlz.pdb /MERGE:.edata=.rdata")
 else()
-    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Werror -Wno-unknown-pragmas -Wno-multichar")
-    set(CMAKE_C_FLAGS_RELWITHDEBINFO "${CMAKE_C_FLAGS} -Ofast -Wall -Werror -Wno-multichar")
+	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Werror -Wconversion -Wno-sign-conversion -Wno-unknown-pragmas -Wno-multichar")
```

Comment:
Can you fix the spacing? See the diff. Thanks!

Reply:
Amended commit to fix.